docs/developer-tools/runtime/runtime-protocol-reference/runtime-protocol-reference.md

```diff
@@ -188,7 +199,7 @@ If you are using Haskell, use the **[Marlowe client library](https://github.com/
 ## Messaging behavior
 
 On a functional level, the Marlowe Runtime protocol multiplexes the four sub-protocols into one. 
-The client always sends one of these seven message types (`RunMarloweSync`, `RunMarloweHeaderSync`, `RunMarloweQuery`, `RunTxJob`, `RunMarloweLoad`, `RunContractQuery`, `RunMarloweTransfer`) to start. 
+The client always sends one of these seven message types (`RunMarloweSync`, `RunMarloweHeaderSync`, `RunMarloweBulkSync`, `RunMarloweQuery`, `RunTxJob`, `RunMarloweLoad`, `RunContractQuery`, `RunMarloweTransfer`) to start. 
```

Comment:
I think this should say "eight" message types.

Reply:
```suggestion
The client always sends one of these eight message types (`RunMarloweSync`, `RunMarloweHeaderSync`, `RunMarloweBulkSync`, `RunMarloweQuery`, `RunTxJob`, `RunMarloweLoad`, `RunContractQuery`, `RunMarloweTransfer`) to start. 
```